Source/OnlineSubsystemEpic/Private/OnlineIdentityInterfaceEpic.h

```diff
@@ -81,10 +81,7 @@ class FOnlineIdentityInterfaceEpic
 	FOnlineSubsystemEpic* subsystemEpic;
 
 	/** Ids mapped to locally registered users */
-	//TMap<int32, TSharedPtr<const FUniqueNetId>> userIds;
-
-	/** Ids mapped to locally registered users */
-	TMap<FUniqueNetIdEpic, TSharedRef<FUserOnlineAccountEpic>> userAccounts;
+	TMap<FString, TSharedRef<FUserOnlineAccountEpic>> userAccounts;
```

Comment:
using the FUniqueNetIdEpic as the key was causing access violation errors during deconstruction/deactivation, swapping it to use the FString representation fixed that issue

Reply:
I think the reason is, that while the methods only accept plain C++ references, the calling methods use shared pointers. Using strings worked, as we copied the data and therefore were free of any potential memory issues.

We also can't simply use the `FUniqueNetId`'s string representation as this is intended to be platform specific ans thus might not be portable.